### PR TITLE
Update SFR Direction

### PR DESCRIPTION
**Pull request by n-kai**
Complete "6.2. PP_MD_V3.3 Security Functional Requirements Direction" except adding EA for modified SFR to BIOSD

### CODE REVIEW

**Comment by woodbe** [comment]
My update also included a few changes for the extended component definitions where we had made updates to the main SFR but not the matching ECD change, so they are included here.

I did not make any changes to the BIOSD as I didn't see any actual change from what had been added for the FPT_KST_EXT requirements as we aren't making any further dependencies on the MDF that we need to do.

**Review by woodbe — CHANGES_REQUESTED** [review]

File: Protection Profile/BiocPP.adoc [hunk 1]
```diff
@@ -378,13 +378,17 @@ Full evaluation activities are not repeated in the <<BIOSD>> for the requirement
...
 ====
-TODO: Need to add additional EAs for modified and additional SFR to <<BIOSD>>
+TODO: Need to add additional EAs for modified SFR to <<BIOSD>>
 ====
 
 ==== Modified SFRs
 
 The SFRs listed in this section are defined in the <<PP_MD_V3.3>> and relevant to the secure operation of the biometric enrolment and verification. It is necessary for the ST author to complete selections and/or assignments for these SFRs in a specific manner in order to ensure that the functionality provided by the mobile device is consistent with the functionality required by the biometric enrolment and verification in order for it to conform to this PP-Module.
 
+===== Class: Cryptographic Support (FCS)
```
[the remark this answers is not in the document]
Comment (n-kai) [reply Protection Profile/BiocPP.adoc:383]: I agree with the first to remove text for key hierarchy.

I added "all keys created upon successful biometric enrolment and verification must be generated, derived, combined, released and destroyed according to SFRs in this class" to address the following NIAP comment. 

Section 8.3.3 ,FPT_BDP_EXT.3.1: This comment is more on the SD activity. This activity talks about encryption, and in terms of NIAP PP-Modules and PPs we require any cryptographic requirements to be specified. The SD currently indicates that the vendor gives a list of cryptographic mechanisms used--this is not acceptable. **This PP should** either specify what is required (ideally through either optional requirements or selection-based requirements) and/or **point to requirements in the Base PP**.

Biometric system (TOE for PPM) needs to generate a KEK, unlock this KEK upon the successful biometric verification and decrypt DEK to unlock the device. So key generation for KEK shall meet FCS_CKM_EXT.3 in MDFPP and key decryption shall meet FCS_COP.1 (or add additional SFRs to MDFPP if the key generation or decryption use different algorithm that are not defined in MDFPP) and .vendor needs to point crypto functions the biometirc system uses to crypt requirement in MDFPP like FCS_CKM_EXT.3 in BMD or KMD.

File: Protection Profile/BiocPP.adoc [hunk 2]
```diff
@@ -393,18 +397,17 @@ The SFRs listed in this section are defined in the <<PP_MD_V3.3>> and relevant t
 
 ===== FPT_KST_EXT.2 No Key Transmission [[FPT_KST_EXT.2]]
 
-*FPT_KST_EXT.2.1*:: The TSF shall not transmit any plaintext key material *or biometric data* outside the security boundary of the TOE.
+*FPT_KST_EXT.2.1*:: The TSF shall not transmit any plaintext key material *and biometric data* outside the security boundary of the TOE.
```
[the remark this answers is not in the document]
Comment (n-kai) [reply Protection Profile/BiocPP.adoc:395]: I agree and I will update it later.

[the remark this answers is not in the document]
Comment (n-kai) [reply Protection Profile/BiocPP.adoc:395]: I am looking at wrong branch. You have done this by another PR

File: Protection Profile/BiocPP.adoc [hunk 3]
```diff
@@ -393,18 +397,17 @@ The SFRs listed in this section are defined in the <<PP_MD_V3.3>> and relevant t
...
 
-*Application Note {counter:remark_count}*:: This SFR is functionally identical to what is defined in the <<PP_MD_V3.3>> with the addition of biometric data as plaintext key materials that must not be transmitted off-device.
+*Application Note {counter:remark_count}*:: This SFR is functionally identical to what is defined in the <<PP_MD_V3.3>> with the addition of biometric data as plaintext key materials that must not be transmitted off-device. 
 
-==== Additional SFRs
+===== Class: Identification and Authentication (FIA)
 
-This section lists additional SFRs that must be added to the TOE boundary in order to implement the functionality in any PP-Configuration where the <<PP_MD_V3.3>> is claimed as the Base-PP.
+This PP-Module does not modify SFRs in FIA class as it is defined in the <<PP_MD_V3.3>>. However, if the TOE claims conformance to a PP-Configuration that includes this PP-Module, only eye, face, fingerprint or vein must be selected in the <<PP_MD_V3.3>> and the same modality must also be selected in this PP-Module.
```
[the remark this answers is not in the document]
Comment (n-kai) [reply Protection Profile/BiocPP.adoc:401]: OK I agree

File: Protection Profile/BiocPP.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (woodbe) [reply Protection Profile/BiocPP.adoc:383]: Yeah, I came around to this, so it is good.

File: Protection Profile/BiocPP.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (woodbe) [reply Protection Profile/BiocPP.adoc]: new commit with this removed

File: Protection Profile/BiocPP.adoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (woodbe) [reply Protection Profile/BiocPP.adoc]: edited here as well (to make sure it doesn't get mixed up later).

**Review by woodbe — APPROVED** [review]

**Pull request merged**